ts/packages/anchor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coral-xyz/anchor",
```

Comment:
Should we do zeta naming now? Probs just ctrl+f "coral" in the whole repo to double check